### PR TITLE
[ASM] - Expander - Update Cortex ASM - Service Ownership playbook

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - Reviewing with @kball-pa

## Related Issues
fixes: [EXPANDR-3830](https://jira-hq.paloaltonetworks.local/browse/EXPANDR-3830) to address #25927

## Description
- Remove the service owner field change related to service accounts in GCP enrichment playbook.
- Remove the service accounts look up in Service Ownership playbook.
- Fix an issue where GCP project is not found.
- Update the Cortex ASM - Service Ownership playbook to check for GCP as an external service before completing GCP related tasks for ranking users.

## Screenshots
Cortex_ASM_-_Service_Ownership_updated_Fri_May_05_2023

Cortex_ASM_-_GCP_Enrichment_ACES-2371_Fri_May_05_2023

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by BigEasyJ** [comment]
cc: @johnnywilkes @capanw @kball-pa @ptoman-pa

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml [hunk 1]
```diff
@@ -464,58 +242,129 @@ tasks:
...
+  "16":
+    id: "16"
+    taskid: 05ce1338-5084-49af-8745-a83af6781f93
+    type: regular
+    task:
+      id: 05ce1338-5084-49af-8745-a83af6781f93
+      version: -1
+      name: Get external service information
+      description: Get service details according to the service ID.
+      script: Cortex Attack Surface Management|||asm-get-external-service
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml:343]: This command is already run in "Cortex ASM - Enrichment".  So we can do one of the following:

- Pass full results of ${ASM.ExternalService.externally_detected_providers} into this playbook as an input (preferred)
- Add a conditional to make sure CASM integration is enabled (similar to "Is Cortex ASM enabled?" task in "Cortex ASM - Enrichment".

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml [hunk 2]
```diff
@@ -6,10 +6,10 @@ starttaskid: "0"
 tasks:
```
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml:6]: Please make sure to update this README after all changes are done: https://github.com/demisto/content/blob/master/Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership_README.md

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
Few small changes needed

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_10.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
```
[the remark this answers is not in the document]
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_10.md:1]: I believe the bug fix should go in first but we can discuss.

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml (hunk 1 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml:343]: I was thinking about this repetition as well. The first option should be a later change, just to keep the main playbook out of scope. I'll go with the second option.

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml (hunk 2 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml:6]: [6e27a2f](https://github.com/demisto/content/pull/26334/commits/6e27a2fbc533642e2118f899ab3b10bf3a760ac6)

File: Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml (hunk 1 above)
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Service_Ownership.yml:343]: [6e27a2f](https://github.com/demisto/content/pull/26334/commits/6e27a2fbc533642e2118f899ab3b10bf3a760ac6)

File: Packs/CortexAttackSurfaceManagement/pack_metadata.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
     "name": "Cortex Attack Surface Management",
     "description": "Content for working with Attack Surface Management (ASM).",
     "support": "xsoar",
-    "currentVersion": "1.6.9",
+    "currentVersion": "1.6.10",
```
[the remark this answers is not in the document]
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/pack_metadata.json]: [6e27a2f](https://github.com/demisto/content/pull/26334/commits/6e27a2fbc533642e2118f899ab3b10bf3a760ac6)

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_10.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_10.md]: [6e27a2f](https://github.com/demisto/content/pull/26334/commits/6e27a2fbc533642e2118f899ab3b10bf3a760ac6)

File: Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Service_Ownership.png
Comment (johnnywilkes) [inline Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Service_Ownership.png:1]: @BigEasyJ , I think you need to update the screenshot after your changes

**Review by johnnywilkes — CHANGES_REQUESTED** [review]
One last small change and we should be good

File: Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Service_Ownership.png
Comment (BigEasyJ) [reply Packs/CortexAttackSurfaceManagement/doc_files/Cortex_ASM_-_Service_Ownership.png:1]: Good catch. Updated in 8fd727.

**Review by johnnywilkes — APPROVED** [review]
Approved

**Review by johnnywilkes — APPROVED** [review]
Approved

**Review by GuyAfik — APPROVED** [review]
looks good, great work!

**Pull request merged**